src/test/ui/parser/issue-87086-colon-path-sep.stderr

```diff
@@ -0,0 +1,50 @@
+error: expected one of `=>`, `@`, `if`, or `|`, found `:`
+  --> $DIR/issue-87086-colon-path-sep.rs:13:12
+   |
+LL |         Foo:Bar => {}
+   |            ^
+   |            |
+   |            expected one of `=>`, `@`, `if`, or `|`
+   |            help: maybe write a path separator here: `::`
```

Comment:
Shouldn't it be `suggestion` rather than `help`?

Reply:
No. I am calling `span_suggestion()`, and this is what it produces.

Reply:
I originally made this use `suggestion:` instead of `help:` but it was considered too verbose...